### PR TITLE
add new footer design

### PR DESCRIPTION
**Pull request by Ross-Clark**

### CODE REVIEW

**Comment by AdamChrimes** [comment]
@Ross-Clark can we add logic to the footer markup to determine whether to use the 3 column layout or just the regular layout? 

For example: If I only add 1 link to the footer, the 3 column layout is applied so it doesn't look quite right:

Screenshot 2019-05-20 at 11 54 58

If there is only 1 link it should just use the regular markup https://nhsuk.github.io/nhsuk-frontend/components/footer/index.html, however, is there more than 5 links in the footer (hard to guessimate because its more about the length of the text rather than the number of items but 5 might be a good start) then use the 3 column layout.

**Comment by Ross-Clark** [comment]
> @Ross-Clark can we add logic to the footer markup to determine whether to use the 3 column layout or just the regular layout?
> 
> For example: If I only add 1 link to the footer, the 3 column layout is applied so it doesn't look quite right:
> 
> Screenshot 2019-05-20 at 11 54 58
> 
> If there is only 1 link it should just use the regular markup https://nhsuk.github.io/nhsuk-frontend/components/footer/index.html, however, is there more than 5 links in the footer (hard to guessimate because its more about the length of the text rather than the number of items but 5 might be a good start) then use the 3 column layout.

I will add the logic as you suggested. I will also have a think about the text lengths of the links and how we can rather tell the users lengths outside range x may not be displayed correctly or add some more logic in to determine it off of text length rather than links, as you mentioned above

**Comment by AdamChrimes** [comment]
Hey @Ross-Clark, do you have any capacity to finish this footer off? I think we discussed having a toggle for the column style footer.

So by default it will be: https://github.com/nhsuk/nhsuk-frontend/tree/master/packages/components/footer#footer-1

But then there is an option to enable the column variant: https://github.com/nhsuk/nhsuk-frontend/tree/master/packages/components/footer#footer-with-list-columns

I think this may be safer than automatically trying to calculate length and items.

**Comment by AdamChrimes** [comment]
I've spoken to Paul, he is going to have a look 👏

**Comment by Ross-Clark** [comment]
I think all that needs to be done is to add a Boolean field into the footer settings and replace the logic that is currently there, where if more than x links are input it changes format, to if the boolean field is true or false.
Would have done it earlier today but I have been in a workshop all day

**Comment by Ross-Clark** [comment]
Did the changes i mention above

**Comment by AdamChrimes** [comment]
@Ross-Clark does this need rebasing? There seems to be changes from somewhere else included now

File: wagtailnhsukfrontend/static/wagtailnhsukfrontend/css/fixes.css [hunk 1]
```diff
@@ -4,3 +4,19 @@
 div.rich-text :last-child {
   margin-bottom: 0;
 }
+
```
Comment (AdamChrimes) [inline wagtailnhsukfrontend/static/wagtailnhsukfrontend/css/fixes.css]: This CSS is now in the NHS.UK frontend library, is it possible to use the latest version (2.3.0) in this change?

Comment (Ross-Clark) [reply wagtailnhsukfrontend/static/wagtailnhsukfrontend/css/fixes.css]: Will add a commit updating the version
The fix grey panel commit must have been from some work I was looking at previously in this repo, it still needs to be put in but i will add it as a seperate pull request

**Pull request merged**